src/models/user.js

```diff
@@ -0,0 +1,15 @@
+import mongoose from 'mongoose'
+
+const userSchema = new mongoose.Schema({
+    firstName: String,
```

Comment:
use following code for defining property for User Schema.

`firstName {
 type: String,
 required: false,
 trim: true.
}`